src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -565,7 +567,20 @@ def test_process_declarations_kind_new_param(f2008_parser):
             "'(KIND = blah)'" in str(err.value))
 
 
-@pytest.mark.xfail(reason="Kind parameter declarations not supported - #585")
+def test_process_declarations_expression_param(f2008_parser):
+    ''' Test that process_declarations handles the kind attribute when
+    it specifies a previously-declared symbol.
+
+    '''
+    fake_parent = KernelSchedule("dummy_schedule")
+    processor = Fparser2Reader()
+    reader = FortranStringReader("real, parameter :: twopi = 2 * pi")
```

Comment:
Isn't this a repeat of the test at line 358?

Reply:
True. I removed this test now.